packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -172,7 +171,14 @@ function getBabelEsmPlugin(config) {
 
 function isProd(config) {
 	let limit = 200 * 1000; // 200kb
-
+	const { src } = config;
+	let swPath = join(__dirname, '..', '..', '..', 'sw', 'sw.js');
+	const userSwPath = join(src, 'sw.js');
```

Comment:
Just curious, can we move the `sw.js` to be in the root folder (`/`) instead of `src`? (just a user preference I guess)

Reply:
everything related to code should be inside `src` right?